messages/CMakeLists.txt

```diff
@@ -56,8 +58,7 @@ add_custom_command(
   # Using prost-build the normal way as part of build.rs does not work due to a cargo bug:
   # https://github.com/danburkert/prost/issues/344#issuecomment-650721245
```

Comment:
Repo moved, the comment is now here: https://github.com/tokio-rs/prost/issues/344#issuecomment-650721245

It seems the relevant [bug](https://github.com/rust-lang/cargo/issues/7915) has been fixed and [stabilized in 1.51](https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#features).

I think this means we can delete the prost-build altogether and use prost as it is documented, without using this custom tool as a workaround.

Reply:
Yeah, I made a new PR.